metabrainz/payments/receipts.py

```diff
@@ -30,11 +30,11 @@ def send_receipt(email, date, amount, name, is_donation, editor_name=None):
             "taxes under section 170 of the Internal Revenue Service code.\n\n"
             "Please save a printed copy of the attached PDF receipt for your records."
         ) % name
-        from_addr = 'donations@' + current_app.config['MAIL_FROM_DOMAIN'],
+        from_addr = 'donations@' + current_app.config['MAIL_FROM_DOMAIN']
```

Comment:
This was the actual issue. `from_addr` became a tuple, which broke the header generation.